youtube_dl/extractor/tastytrade.py

```diff
@@ -33,7 +49,36 @@ def _real_extract(self, url):
             r'data-media-id=(["\'])(?P<code>(?:(?!\1).)+)\1',
             webpage, 'ooyala code', group='code')
 
-        info = self._search_json_ld(webpage, display_id, fatal=False)
+        info = {'id': None, 'title': None, 'description': None}
+
+        json_ld_info = self._search_json_ld(
+            webpage, display_id, default=None, fatal=False)
+
+        if (json_ld_info):
+            info = json_ld_info
+        else:
+            escaped_json_string = self._search_regex(
+                r'var episodeData = \$.parseJSON\("(?P<episode_json>.*)"\)',
+                webpage,
+                'episode json',
+                fatal=False,
+                group='episode_json'
+            )
+
+            if (escaped_json_string):
+                if sys.version_info[0] >= 3:
+                    unescaped_json_string = bytes(
+                        escaped_json_string, "utf-8").decode('unicode_escape')
+                else:
+                    unescaped_json_string = escaped_json_string.decode(
+                        'string_escape')
+                metadata = self._parse_json(unescaped_json_string, ooyala_code)
+                info = {
+                    'id': metadata.get('mediaId'),
```

Comment:
No.

Reply:
I'm not sure exactly what's meant by this comment. Could you provide more context?

Reply:
You are using different ids in url and here. Don't do that.

Reply:
Ah, okay. Thanks for the clarification.